ivy/functional/frontends/numpy/mathematical_functions/trigonometric_functions.py

```diff
@@ -11,6 +11,28 @@
 )
 
 
+@handle_numpy_out
+@handle_numpy_dtype
+@to_ivy_arrays_and_back
+@handle_numpy_casting
+@from_zero_dim_arrays_to_scalar
+def _modf(
```

Comment:
Can you please also clarify why do you need that underscore here? Thanks
```suggestion
def modf(
```

Reply:
I referred to the ivy frontends docs which state that "In NumPy, add() is categorised under mathematical_functions with a sub-category of arithmetic_operations as shown in the [numpy mathematical functions](https://numpy.org/doc/stable/reference/index.html) directory. It is important to note that add() is a universal function ([ufunc](https://numpy.org/doc/stable/reference/ufuncs.html)) in NumPy, thus the function is actually an object with instance methods like .at and .reduce, etc. We deal with this in the NumPy frontend by including a ufunc class and initialising it in the __init__ file" so thats why I used _modf.

https://lets-unify.ai/ivy/deep_dive/ivy_frontends.html